templatestore/frontend/.eslintrc

```diff
@@ -14,7 +14,6 @@
       }
    },
    "rules": {
-       "indent": ['error', 4],
-       "space-before-function-paren": ["error", "never"],
+       "space-before-function-paren": ["error", "never"]
```

Comment:
why this was removed ? 
Any particular reason for it ?

Reply:
I added it for local dev, then removed it.
Should not have pushed it.